trestlebot/tasks/sync_oscal_content_task.py

```diff
@@ -148,16 +179,63 @@ def write_ordered_data_to_yaml(
             yaml.indent(mapping=4, sequence=4, offset=4)
         yaml.dump(data, file_path)
 
-    def _update_selections_rules_in_memory(
-        self, rule_list: List[str], add_params: bool = True
+    def _update_control_file_change_in_memory(
+        self, cac_control: Dict[str, Any], oscal_control: ImplementedRequirement
+    ) -> None:
+        """
+        In memory update cac control file changes
+        """
+        rule_list = populate_if_dict_field_not_exist(cac_control, "rules", [])
+
+        removed_variable = []
+        cac_rule_list = []
+        for rule_index, rule in enumerate(rule_list):
+            if "=" in rule:
+                # variable
+                v_id, v_value = rule.split("=")
```

Comment:
`v_value` is not used here. Maybe the value could be ignored: `v_id, _ = rule.split("=")`

Reply:
Updated